easybuild/easyblocks/t/tensorflow.py

```diff
@@ -461,6 +465,19 @@ def configure_step(self):
         # note that this may be an RPATH wrapper script (when EasyBuild is configured with --rpath)
         ld_path = which('ld')
         self.binutils_bin_path = os.path.dirname(ld_path)
+        if self.toolchain.is_rpath_wrapper(ld_path):
+            if os.path.basename(os.path.dirname(os.path.dirname(ld_path))) == RPATH_WRAPPERS_SUBDIR:
+                # TF expects all binutils binaries in a single path but newer EB puts each in their own subfolder
+                # Add symlinks to each binutils binary into a single folder
+                new_rpath_wrapper_dir = os.path.join(self.wrapper_dir, RPATH_WRAPPERS_SUBDIR)
+                binutils_bin_path = os.path.join(get_software_root('binutils'), 'bin')
```

Comment:
The assumption here that `binutils` is provided as a build dependency is not always correct, since it may be filtered out as dependency (which we do in [EESSI](https://eessi.io)).

So this definitely requires a follow-up PR which has some kind of fallback mechanism in case `get_software_root('binutils')` produces `None`.

Reply:
see also #2218, the problem being fixed there is essentially being introduced again here